src/main/test/ApplicationUtilsTests.cpp

```diff
@@ -658,4 +658,42 @@ TEST_CASE("application major version numbers", "[applicationutils]")
     CHECK(getStellarCoreMajorReleaseVersion("v19.1.2-10") == std::nullopt);
     CHECK(getStellarCoreMajorReleaseVersion("v19.9.0-30-g726eabdea-dirty") ==
           std::nullopt);
+}
+
+TEST_CASE("standalone quorum intersection check", "[applicationutils]")
+{
+    Config cfg = getTestConfig();
+    const std::string JSON_ROOT = "testdata/check-quorum-intersection-json/";
```

Comment:
This fails to open the file if stellar-core is not run from src/

Reply:
Fixed in 3f6f7e65dc8477b872e6db8e7c7c6bf8113ca467 by inlining the JSON into the tests.

Reply:
we run tests from `src` normally anyways, as other tests rely on `testdata` folder, so I think this is fine. Having some test json files is helpful since they can also be used to manually test the command from command line, so I'd recommend we keep them.

Reply:
That makes sense, it's nice to have the JSON files. I reverted the change that inlined the tests.